libs/IO/ImageTIFF.cpp

```diff
@@ -413,22 +412,37 @@ HRESULT CImageTIFF::ReadHeader()
 			((photometric != 2 && photometric != 1) ||
 				(ncn != 1 && ncn != 3 && ncn != 4)))
 			bpp = 8;
-		switch (bpp) {
+
+		bool implemented = true;
+
+		switch (bpp){
 		case 8:
+			if (ncn >= 3){
```

Comment:
pls use the same format as the rest of OpenMVS code

Reply:
I think you mean having a space between `(` and `{` right? (I added that).